docs/learn/bossroom/getting-started-boss-room.md

```diff
@@ -6,200 +6,275 @@ description: Learn more about installing and running the Boss Room game sample.
+
+```json
+"com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop"
+```
 
-1. Open your Unity Hub.
-1. Click the dropdown arrow next to **Open**, then select **Add project from disk**.
-1. Select the root folder of the downloaded project. For example, `com.unity.multiplayer.samples.coop-<version>`.
+For more information on the art in Boss Room, see [ART_NOTES.md](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/Documentation/ART_NOTES.md).
 
-:::note
-The first time you open the project, Unity will import all assets, which will take longer than usual - this is normal. The Unity Netcode for GameObjects package will also be installed with Boss Room.
+### Getting the project
 
-**Issues with importing due to parental control software**: If you have issues with importing you may want to check your DNS settings as some ISP parental controls  may block GitHub access. For example, see this information on [WebSafe](https://community.virginmedia.com/t5/Networking-and-WiFi/Web-Safe-Breaks-GitHub/td-p/4279652).
-:::
+You can get the project by downloading it directly or cloning locally. However, if you use git, you must install Git LFS.
 
-## Register the project with Unity Gaming Services (UGS)
+#### Direct download
 
-This project leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must first [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the **Unity Dashboard**, and enable both the [Relay](https://docs.unity.com/relay/get-started.html) and [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services.
+You can download the latest version of Boss Room from our [Releases](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page.
 
-## Open the project
+Alternatively, you can click the green **Code** button, then select **Download Zip**. This downloadd the branch that you are currently viewing on Github.
+
+:::note
+**Note for Windows users**: Using Windows' built-in extraction tool may generate an `Error 0x80010135: Path too long` error window which can invalidate the extraction process. A workaround for this is to shorten the zip file to a single character (for example, "`c.zip`") and move it to the shortest path on your computer (most often right at `C:\`) and retry. If that solution fails, another workaround is to extract the downloaded zip file using [7zip](https://www.7-zip.org/).
+:::
 
-To open the project for the first time:
+#### Installing Git LFS to clone locally
```

Comment:
Since we make note about Git LFS under "Getting the project", would it make sense to migrate this section above the direct download? (if so then you could ignore the comment about adding the link mentioned above)

Reply:
Hmm, yeah, that would make more sense. I'll do that.

Reply:
Just to be sure of understanding, direct download doesn't require LFS though.